cli/templatecreate_test.go

```diff
@@ -241,6 +241,24 @@ func TestTemplateCreate(t *testing.T) {
 		err = create()
 		require.NoError(t, err, "Template must be recreated without error")
 	})
+
+	t.Run("WithParameterExceedingCharLimit", func(t *testing.T) {
+		t.Parallel()
+		client := coderdtest.New(t, &coderdtest.Options{IncludeProvisionerD: true})
+		coderdtest.CreateFirstUser(t, client)
+		cmd, root := clitest.New(t, "templates", "create", "1234567890123456789012345678901234567891", "--test.provisioner", string(database.ProvisionerTypeEcho))
+		clitest.SetupConfig(t, client, root)
+		pty := ptytest.New(t)
+		cmd.SetIn(pty.Input())
+		cmd.SetOut(pty.Output())
```

Comment:
I don't believe a pty is necessary here since you're not using the output of it

Reply:
Ah right, I'll move!